tooling/prettier-config/README.md

```
@@ -0,0 +1,58 @@
+pnpm add --save-dev prettier
+```
+
+### With Yarn
+
+```sh
+yard add --dev prettier
+```
+
+Next, install `@wso2/prettier-config`:
+
+### With npm
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)